common/common_test.go

```diff
@@ -950,3 +953,70 @@ func TestTimeFromUnixTimestampFloat(t *testing.T) {
 		t.Error("Test failed. Common TimeFromUnixTimestampFloat. Converted invalid syntax.")
 	}
 }
+
+func TestGetDefaultDataDir(t *testing.T) {
+	expectedOutput := os.Getenv("APPDATA") + GetOSPathSlash() + "GoCryptoTrader"
```

Comment:
Last request here is that we'll need to add test support for darwin/linux, then this PR looks GTG!